README.md

```diff
@@ -288,29 +288,34 @@ Basic proof of 'ipfs working' locally:
 
 SUBCOMMANDS
   BASIC COMMANDS
-    init          Initialize ipfs local configuration
-    add <path>    Add a file to ipfs
-    cat <ref>     Show ipfs object data
-    get <ref>     Download ipfs objects
+    init          Initialize local IPFS configuration
+    add <path>    Add a file to IPFS
+    cat <ref>     Show IPFS object data
+    get <ref>     Download IPFS objects
```

Comment:
I feel like we should add, that the file/object isn't only fetched to the local node, but also stored in the current working directory in a new folder which is named after the CID.

Reply:
Mind opening a separate PR with your suggestions?
I'd like  to keep this one scoped to known fixes  and avoid bikeshed :)